var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -95,8 +95,8 @@ class Gdal(AutotoolsPackage):
     # Required dependencies
     depends_on('libtiff@3.6.0:')  # 3.9.0+ needed to pass testsuite
     depends_on('libgeotiff@1.2.1:')
-    depends_on('libgeotiff@:1.4', when='@:2.4')
-    depends_on('libgeotiff@1.5:', when='@3:')
+    depends_on('libgeotiff@:1.4', when='@:2.4.0')
+    depends_on('libgeotiff@1.5:', when='@2.4.1:')
```

Comment:
Are you sure? According to https://trac.osgeo.org/gdal/wiki/Release/3.0.0-News#GDALOGR3.0.0-GeneralChanges this change was made in 3.0.0. There's no mention of changing the required libgeotiff version in the 2.4.1 or 2.4.2 release notes.

Reply:
Sorry - still learning spack version magic. 2.4.1 is compatible with libgeotiff 1.5, but requires 1.2.1+. Is it better now?